nitter/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_NITTER_IP
+      APP_PORT: 8080
+
+  web:
+    # Official Nim image used as base image for nitter doesn't offer Arm variant so using a custom image w/ patch for umbrel
+    # https://github.com/ceramicwhite/umbrel-patch-ci/blob/master/nitter/umbrel.patch
+    image: ceramicwhite/nitter:build-20220906@sha256:6ddb53fe66846ffcded344d1919bee3149b705f04a9347c37b8eaa562695ce81
+    restart: on-failure
+    stop_grace_period: 1m
+    init: true
+    # app is run with su-exec to drop privileges to a non-root user
+    #user: "1000:1000"
```

Comment:
I took a look at the source and your patches today. I'm wondering if we can with reletive ease enable an arbitrary user/group ID to be used.

My idea is; we drop these lines: https://github.com/ceramicwhite/umbrel-patch-ci/blob/090560766cd67f141ec4147bb90dc68fcff42d3e/nitter/umbrel.patch#L82-L83

And then we drop the `chown` lines e.g. `chown www-data:www-data ABC` (https://github.com/goodtiding5/docker-nitter/blob/01a2e49c7a050edbe4b06bd49bb97b12fcac2a17/entrypoint.sh#L27).

@ceramicwhite Thoughts on this?

Reply:
Yeah you're right it's fine on my end now